WcaOnRails/app/views/polls/edit.html.erb

```diff
@@ -6,15 +6,11 @@
 
   <% is_actually_confirmed = Poll.find(@poll.id).confirmed? %>
   <% if is_actually_confirmed %>
-    <% if @poll.over? %>
-      <div class="alert alert-danger">
-        This poll is closed (deadline has passed). Check out the <%= link_to "results", polls_results_path(@poll) %>.
-    </div>
-    <% else %>
-      <div class="alert alert-info">
-        This poll is open to <%= link_to "voting", polls_vote_path(@poll) %>. Editing question or options is not allowed.
-      </div>
-    <% end %>
+    <%= if @poll.over?
+          alert :danger, "This poll is closed (deadline has passed). Check out the #{link_to "results", polls_results_path(@poll)}"
```

Comment:
I understand that you were looking to get this down to one line, but I think would be cleaner (and safer!) as a block.